FaunaDB.Client.Test/ClientTest.cs

```diff
@@ -894,7 +894,7 @@ await client.Query(Paginate(Events(Singleton(createdInstance))))
             // Deprecated
 
             Assert.AreEqual(BooleanV.True,
-                await client.Query(Contains(Path("favorites", "foods"), favoritesObj)));
+                await client.Query(ContainsPath(Path("favorites", "foods"), favoritesObj)));
```

Comment:
I think we should keep `Contains` and duplicate it w/ `ContainsPath` as we do still "accept" the `Contains` expression. What do you think? 
Is there a way to tell the compiler we are ok w/ that warnings for that block ?

Reply:
```
#pragma warning disable 618

Contains blah

#pragma warning restore 618
```

does that work ?

Reply:
Sure. Agree. I'll duplicate tests for Contains and ContainsPath and suppress warning for contains (no way to resolve it, because Contains is obsolete)

Reply:
> ```
> #pragma warning disable 618
> 
> Contains blah
> 
> #pragma warning restore 618
> ```
> 
> does that work ?

#pragma warning disable CS0618
sure. that  works